test/dsl_test.go

```diff
@@ -172,7 +172,7 @@ func (o *opt) close() {
 func (o *opt) runInitOnce() {
 	o.initOnce.Do(func() {
 		o.clock = &libkbfs.TestClock{}
-		o.clock.Set(time.Unix(0, 0))
+		o.clock.Set(time.Unix(1, 0))
```

Comment:
I'm curious why this is needed?

Reply:
It's needed because otherwise the ToTime+FromTime trick strip turns the timestamp into a zero timestamp, and we have checks for that in the library (to make sure an `ImmutableRootMetadata` never has an empty timestamp, for example).